### PR TITLE
Fix mongo setup

### PR DESCRIPTION
**Pull request by hovoodd**

### CODE REVIEW

File: docker-compose.test.yml [hunk 1]
```diff
@@ -1,40 +1,52 @@
...
     ports:
-      - "27018:27017"
+      - 27018:27017
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
+  mongo-replicator-test:
+    container_name: ship-test-mongo-replicator
+    build:
+      context: ./mongo-replicator
+      dockerfile: Dockerfile-test
```
Comment (SobakaSlava) [inline docker-compose.test.yml:17]: May be `Dockerfile.test`? As we have for `Dockerfile.dev`

File: src/config/environment/test.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "mongo": {
-    "connection": "mongodb://mongo-test:27017/api-test?replicaSet=rs-test"
+    "connection": "mongodb://mongo-test:27017/api-test?replicaSet=ship-test-rs"
```
Comment (NesterenkoNikita) [inline src/config/environment/test.json:3]: Please, update other values as well

File: src/migrations/migrations-log/migration-log.schema.js [hunk 3]
```diff
@@ -1,13 +1,10 @@
 const schema = {
   $jsonSchema: {
-    required: ['companyId', 'startTime', 'finishTime', 'status'],
+    required: ['startTime', 'status'],
```
Comment (NesterenkoNikita) [inline src/migrations/migrations-log/migration-log.schema.js:3]: `migrationVersion` required?

File: src/config/environment/test.json (hunk 2 above)
Comment (hovoodd) [reply src/config/environment/test.json:3]: What values?

Comment (NesterenkoNikita) [reply src/config/environment/test.json:3]: @hovoodd ports

**Pull request merged**